app.rb

```diff
@@ -91,4 +98,32 @@ def list_rentals_for_person
       puts "Date: #{rental.date}, Book: #{rental.book.title}"
     end
   end
+
+  def run # rubocop:disable Metrics/MethodLength, Metrics/CyclomaticComplexity
+    loop do
+      display_options
+      option = gets.chomp.to_i
+      if option == 7
+        puts 'Thank you for using the app. Goodbye!'
+        break
+      elsif option.between?(1, 6)
+        case option
+        when 1
+          list_books
+        when 2
+          list_people
+        when 3
+          create_person
+        when 4
+          create_book
+        when 5
+          create_rental
+        when 6
+          list_rentals_for_person
```

Comment:
- [ ] Great job on your code refactoring! 😊 However, I noticed a potential issue that affects the functionality of the list rental feature. 

   - 🚀 It appears that when a user tries to list a rental, your code displays a list of all available persons before asking for the `ID` of the person whose rental they want to view. According to the project requirements and the provided video, the user should first input the ID of the person whose rental details they want to see, and then the app should display the available person's rental details. 

    - 💡 To align with the expected behavior, I recommend revisiting this part of your code and ensuring that the user is prompted for the `ID` only without displaying the list of persons available. This adjustment will bring your code in line with the intended functionality as demonstrated in the project requirements video.
    
Happy coding! 😊👨‍💻🚀

Reply:
- Refactor `list_rentals_for_person` method to prompted for the ID only without displaying the list of persons available.

Reply:
- Refactor `list_rentals_for_person` method to prompted for the ID only without displaying the list of persons available.